guides/v2.1/security/google-recaptcha.md

```diff
@@ -36,11 +36,11 @@ php bin/magento setup:upgrade
 
 ## Configure reCAPTCHA
 
-For full details and walk-throughs for configuring and using Google reCAPTCHA, see the [Magento Admin User Guide](https://docs.magento.com/m2/2.1/ee/user_guide/stores/security-google-recaptcha.html).
+See the [Magento Admin User Guide](https://docs.magento.com/m2/ce/user_guide/stores/security-google-recaptcha.html) for configuration options to the Magento Admin UI and storefront.
 
 ## Troubleshooting
 
-If you have issues with reCAPTCHA, you can disable it from the command-line. This will disable the module for backend access.
+The extension supports a command line option for disabling reCAPTCHA and allow backend access. Use this commands when you cannot access the Magento Admin UI.
```

Comment:
for disabling reCAPTCHA --  maybe you can just leave off the backend access piece -- disabling it is probably enough